### ISSUE TITLE
Add filter/tag to display events in the correct order

### ISSUE THREAD

**Issue by qbixx** [transformstudios/statamic-events#59]
Related to #54 

With the change of the `{{ events }}` tag we can show repeating events only once. To list all the events we can use a collection tag to display all the events. However this introduces a new issue. Since we can't use the original setup for the `{{ events }}` tag we need to use a collection tag. We could indicate to sort them on the `start_date` but that won't work to sort the events in the correct way.

When using e.g. the code below, it will display all the recurring events first, since those have the oldest start_date in comparison to events that have only one or multiple dates. Therefore the events aren't sorted on the actual date when the recurring event is.

```
{{ collection:acties sort="start_date" }}
    {{ events:upcoming :event="id" limit="1" }}
         {{ partial:components/action_item }}
    {{ /events:upcoming}}
{{ /collection:acties }}
```

**Comment by edalzell** [comment]
You could maybe make a custom tag that used the same Event methods, then sort the results?

**Comment by qbixx** [comment]
Hi @edalzell, thanks for your reply. Sorry, i'm not really sure what you mean. Could you perhaps point me in the right direction?

**Comment by edalzell** [comment]
> Hi @edalzell, thanks for your reply. Sorry, i'm not really sure what you mean. Could you perhaps point me in the right direction?

Sorry for the delay. I meant that you'd need to create a [custom tag](https://statamic.dev/extending/tags) that used the underlying event methods (upcoming, etc) to achieve what you want.

This isn't functionality that will be added to Events.
